jekyll/_docs/enterprise/advanced-configuration.md

```diff
@@ -0,0 +1,224 @@
+
+Also note that when the instructions ask you if you use a proxy, they will also prompt you for the address. It is **very important** that you input the proxy in the following format `<protocol>://<ip>:<port>`. If you are missing any part of that, then apt-get won't work correctly and the packages won't download. 
+
+If for any reason you can't access the page the CircleCI replicated console, but the services machine seems to be running, try to ssh tunnel into the machine doing the following: `ssh -L 8800:<address you want to proxy through>:8800 ubuntu@<ip_of_services_box>`. 
+
+
+### Builder Box configuration:
+This one is a bit easier - it has few moving parts:
+
+- Plenty of `curl` commands to download CircleCI init-builder script and jars from S3.  If you whitelist S3 traffic, then you're are good.  Otherwise, curl respects environment settings.
+- `awscli` tool to download container from S3. The same rules of `curl` apply here (i.e. dependent on S3 traffic and environment variable)
+- CircleCI JVM.  This makes plenty of connections:
+  - Internal call that should be excluded from http proxy — presumably this happens by default:
+    - Any connections to other builders and/or Services box
+    - Connections to GitHub Enterprise.
+  - Other calls:
+    - Amazon EC2 metadata (http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) .  This **should not** be proxied.  If it is, then the machine will be misconfigured.
+    - Amazon S3 traffic — note S3 discussion above
+    - Amazon EC2 API - EC2 API traffic may need to be proxied.  You would note lots of failures (timeout failures) in logs if the proxy setting is misconfigured, but it will not block CCIE functioning.
+
+### On Builder boxes Launch Configuration - change it to add the /etc/environment modification
+
+```
+#!/bin/bash
+
+(cat <<'EOF'
+# Append
+HTTP_PROXY=10.0.0.33:3128
+HTTPS_PROXY=10.0.0.33:3128
+NO_PROXY=169.254.169.254,10.0.1.238,127.0.0.1,localhost,ghe.sphereci.com
+JVM_OPTS="-Dhttp.proxyHost=10.0.0.33 -Dhttp.proxyPort=3128 -Dhttps.proxyHost=10.0.0.33 -Dhttps.proxyPort=3128 -Dhttp.nonProxyHosts=169.254.169.254|10.0.1.238|127.0.0.1|localhost|ghe.sphereci.com $JVM_OPTS"
+
+EOF
+) | sudo tee -a /etc/environment
+
+set -a
+. /etc/environment
+
+# Back to normal BUT MUST REMOVE sudo
+curl https://s3.amazonaws.com/circleci-enterprise/init-builder-0.2.sh | \
+    CIRCLE_SECRET_PASSPHRASE=<passphrase entered on system console (services box port 8800) settings> \
+    SERVICES_PRIVATE_IP=10.0.1.238 bash
+```
+
+### Installing CircleCI Builder
+This really depends on what type of distro you are installing CircleCI on, but the above instructions cover a basic kind of install.
+
+
+##GPU Install Instructions 
+
+This is only supported in our newest release ( 1.47.0 +). If you need help upgraging, please reachout to enterprise-support@circleci.com
+
+First
```

Comment:
this still needs to be addressed in this revision. I just wanted to commit what I had in the moment to do a less bulky change. As a note these instructions will also be changing to include GPU Docker support.